### ISSUE TITLE
4.x - Add "cover" as a shape option for card component

### ISSUE THREAD

**Issue by bspeare** [uiowa/uids#804]
As a user of the card component, I would like to stretch my card image to fill the vertical space of the card using the "cover" shape option that applies the `object-fit` property. 

# Proposed solution

**Shape**
- Circle
- Widescreen
- Widescreen: cover
- Square
- Square: cover

With padded

 
<img width="1578" alt="Screen Shot 2022-10-20 at 12 48 05 PM" src="https://user-images.githubusercontent.com/1036433/197021822-f850f1b0-50ee-466c-817c-62f5ea1d992c.png">

Without padded

<img width="1574" alt="Screen Shot 2022-10-20 at 12 48 35 PM" src="https://user-images.githubusercontent.com/1036433/197021802-c4444493-63b3-4134-95e4-092d89685d54.png">

You can preview this functionality in https://github.com/uiowa/uids/pull/803.

**Comment by joewhitsitt** [comment]
Do we want to entertain the idea of a widescreen vs square "shape" cover or am I not thinking responsively?

**Comment by bspeare** [comment]
@joewhitsitt Yes, that could be a good idea!

**Comment by joewhitsitt** [comment]
I guess I can visualize the square cover compared to the default cover screenshot above, but I am not sure what a widescreen cover would look like since the height would be determined by the card content.

**Comment by bspeare** [comment]
That's true.  A widescreen ratio quickly becomes something else once the card fills up with text:

<img width="1024" alt="Screen Shot 2022-10-21 at 8 43 49 AM" src="https://user-images.githubusercontent.com/1036433/197210356-23c41649-8ff8-4e5a-9d76-3a028599df4d.png">

Perhaps promising the aspect ratio as part of the cover is not the best solution.
